apps/base/utils.py

```diff
@@ -69,3 +76,24 @@ def decode_data(data):
     for i in data:
         decoded.append(base64.decodestring(i+"=="))
     return decoded
+
+
+def send_sendgrid_email(email="", message=""):
+    """
+    Send email to sendgrid with message
+
+    Keyword Arguments:
+        email {string} -- The email to which the message is to be sent
+        message {string} -- The message which is to be sent to the email
+    """
+    try:
+        sg = sendgrid.SendGridAPIClient(apikey=settings.SENDGRID_API_KEY)
```

Comment:
@RishabhJain2018 @deshraj
What do you think about using `requests` for [sendgrid mail api](https://sendgrid.com/docs/API_Reference/Web_API_v3/Mail/index.html) instead of sendgrid's Library here as it increases dependencies?